code/modules/mob/living/simple_animal/bot/medbot.dm

```diff
@@ -242,8 +242,7 @@
 	if(assess_patient(H))
 		last_found = world.time
 		if((last_newpatient_speak + 300) < world.time) //Don't spam these messages!
-			var/message = pick("Hey, [H.name]! Hold on, I'm coming.","Wait [H.name]! I want to help!","[H.name], you appear to be injured!")

```

Comment:
don't remove these, find a way to sync the words to the sounds (probably just two lists where 1 index lines the sound with the text).

Because you literally can't tell what some of the sounds are saying <i>even if you know the text lines</i>